app/src/main/res/layout/fragment_achievements.xml

```diff
@@ -341,12 +341,13 @@
                 app:layout_constraintLeft_toRightOf="@id/wikidata_edits_icon"
                 app:layout_constraintTop_toTopOf="parent"
                 app:layout_constraintBottom_toBottomOf="parent"
-                app:layout_constraintRight_toLeftOf="@id/images_nearby_info"
+                app:layout_constraintRight_toLeftOf="@id/images_nearby_info_icon"
```

Comment:
I think this may be the cause of test fails, can you try to use "layout_constraintEnd_toStartOf" instead?